src/components/Form.js

```diff
@@ -7,9 +7,11 @@ const Form = ({ addBook }) => {
   const handleSubmit = (e) => {
     e.preventDefault();
 
-    addBook({ title, author });
-    setTitle('');
-    setAuthor('');
+    if (title.trim() !== '' && author.trim() !== '') {
+      addBook({ title, author });
+      setTitle('');
+      setAuthor('');
+    }
```

Comment:
- [ ] When you create a book, you are not creating any `item_id` for the Book being created. This would make all new books created to be deleted when you click a delete button for any of them because this expression:

```JavaScript
removeBook: (state, action) => {
  state.books = state.books.filter((book) => book.item_id !== action.payload);
},
```
The filter expression will result in `book.item_id !== null`. Therefore all other new books will be deleted save the ones created through the static array. Kindly include the `item_id` key for every book. 😎 👍 

![chrome-capture-2023-7-30](https://github.com/mumbereh/bookstore/assets/77106962/a3674f7a-c6f1-4513-9519-e5c42796181e)